packages/viewer/src/lib/plots/BoxPlot.svelte

```diff
@@ -160,7 +160,7 @@
       return () => {
         clientSelection.destroy();
         filter.update({
-          source: clientSelection,
+          source: clientSelection as any,
```

Comment:
It seems like the client returned by `makeClient` causes some problem here. This probably need to be fixed in Mosaic. Temporarily cast to any.